packages/era/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@lodestar/era",
+  "description": "Era file handling module for Lodestar",
+  "license": "Apache-2.0",
+  "author": "ChainSafe Systems",
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/lodestar.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "version": "1.34.1",
```

To keep up with the other package versions.  This will change though once 1.36 is released shortly.
```suggestion
  "version": "1.35.0",
```